src/lib.rs

```diff
@@ -457,15 +466,7 @@ pub mod unsync {
         ///
         /// This method is allowed to violate the invariant of writing to a `OnceCell`
         /// at most once because it requires `&mut` access to `self`. As with all
-        /// interior mutability, `&mut` access permits arbitrary modification:
-        ///
-        /// ```
-        /// use once_cell::unsync::OnceCell;
-        ///
-        /// let mut cell: OnceCell<u32> = OnceCell::new();
-        /// cell.set(92).unwrap();
-        /// cell = OnceCell::new();
```

Comment:
I think the example is where it needs to be, but is incomplete. Let's change it to 

```rust
use once_cell::unsync::OnceCell;

let mut cell: OnceCell<u32> = OnceCell::new();
cell.set(92).unwrap();
*cell.get_mut().unwrap() = 93;
assert_eq!(cell.get(), Some(&93));

```

Reply:
Sounds good. I fixed like that in [a361.](https://github.com/matklad/once_cell/pull/180/commits/a36132d4513f7316576a3b45ddb768210f0825ab)